src/styles/blocks/title.css

```
@@ -0,0 +1,5 @@
+.card__title {
+  margin: 0 0 4px;
```

может лучше так, чтобы по всему доку было одинаково?
```suggestion
  margin-bottom: 4px;
```